### DISCUSSION TITLE
Offshore marine alerts not visible on Alerts API

### DISCUSSION THREAD

**Discussion by pelkoe** [weather-gov/api#670] [Q&A] [answered]
Hi,

I'm not able to find alerts
https://alerts.weather.gov/cap/wwacapget.php?x=MZPI1266675C90D8.TropicalStormWarning.1266675E6854MZPI.NHCOFFPZ8.6c6f8cdf755e15f2b1358a399513a46b
https://alerts.weather.gov/cap/wwacapget.php?x=MZPI1266675C90D8.StormWarning.1266675E6854MZPI.NHCOFFPZ7.75ab9fe26b69c9d838b7da92ec4916ad
and others from NWS National Hurricane Center and NWS Ocean Prediction Center in https://api.weather.gov/alerts/active.

I've read some previous discussions here on the API not apparently distributing data coming from these parties or that there is no support yet for offshore weather alerts. Is this still true? If so, where could I find this data now that the support for CAP v1.1. API is coming to an end at the end of the year?

Thanks,

Eetu

**Comment by dwhitemv25** [comment, marked as answer]
If it helps, the alert linked is a section of the Offshore Waters Forecast product for that region, whose current version appears at https://www.nhc.noaa.gov/text/MIAOFFPZ8.shtml. As a stopgap measure, you can scrape that page, split the message up by UGC codes, and look for warning text in each section. 

For the Eastern Atlantic and Caribbean areas, NHC offers [RSS feeds](https://www.nhc.noaa.gov/aboutrss.shtml) of the forecast, but I didn't find any links to their Pacific counterparts. Might be worth emailing the NHC webmaster and see if they have those hidden somewhere.

**Comment by elusiveuphoria** [comment]
Forgive me for the 2yo bump; but I'm looking for an answer to this exact question. I've noticed this issue about the API service as well and had my own solution to supplement this blindspot, up until the discontinued support of CAP v1.1 (as mentioned). For much of the interim I haven't been fully invested in finding a new solution, but recent projects have led me down this rabbit hole once again and much of what I am finding isn't satisfactory. I've tried emailing NCO ops (the stated official POC for issue reporting for the API) a few times over the years but the one time I got a response, they didn't seem to understand the issue as described. Thankful for any breadcrumbs anyone here can provide.

**Reply by dwhitemv25** [reply]
Please start a new topic, and be specific about what product(s) you are looking for, and why the accepted answer doesn't work for you.
